src/SonarQube.Bootstrapper/SonarQube.Bootstrapper.csproj

```diff
@@ -18,4 +18,15 @@
     <ProjectReference Include="..\SonarQube.TeamBuild.PreProcessor\SonarQube.TeamBuild.PreProcessor.csproj" />
     <ProjectReference Include="..\SonarScanner.Shim\SonarScanner.Shim.csproj" />
   </ItemGroup>
-</Project>
+  <!-- Conditionally reference project only if the framework is classic .NET Framework (not core or standard). -->
+  <ItemGroup Condition="('$(TargetFrameworkIdentifier)' == '.NETFramework')">
+      <ProjectReference Include="..\SonarQube.TeamBuild.Integration.Legacy\SonarQube.TeamBuild.Integration.Legacy.csproj" />
```

Comment:
I decided to create a separate assembly with all the references to 46 TFS libraries. It will be easier to maintain and (hopefully soon) delete.

Reply:
I am not a big fan of the name. Even if we might end up with a lot of legacy things I would still prefer to have them in dedicated libraries. What about `SonarQube.TeamBuild.Integration.XamlBuild`?